Unix/tests/codec/mof/blue/parameterset.c

```diff
@@ -21,6 +21,12 @@
 # define STRLEN wcslen
 #else
 #include <wchar.h>
+
+#if !defined(CONFIG_HAVE_WCSCASECMP)
+// defined in pal/strings
+int wcscasecmp(const wchar_t* s1, const wchar_t* s2);
+#endif
```

Comment:
If this is implemented elsewhere, why do we need the definition here? Can't we grab this from a header file? The manpage for this say to grab definition from `wchar.h`.